### PR TITLE
Harmonize onboarding and empty states

### PR DESCRIPTION
**Pull request by sharvaniharan**
**Phab:** https://phabricator.wikimedia.org/T257841

### CODE REVIEW

File: app/src/main/res/layout/fragment_history.xml [hunk 1]
```diff
@@ -12,31 +12,31 @@
...
+        <ImageView
+            android:id="@+id/history_empty_image"
+            android:layout_width="168dp"
+            android:layout_height="168dp"
+            android:layout_gravity="center_horizontal"
+            android:contentDescription="@null"
+            app:srcCompat="@drawable/no_history" />
+
         <TextView
             android:id="@+id/history_empty_title"
```
Comment (cooltey) [inline app/src/main/res/layout/fragment_history.xml:24]: Should the title have a bold text?

Comment (sharvaniharan) [reply app/src/main/res/layout/fragment_history.xml:24]: Good catch! done!

File: app/src/main/res/layout/fragment_reading_lists.xml [hunk 2]
```diff
@@ -61,15 +61,14 @@
                     style="@style/MaterialLargePrimaryTitle"
```
Comment (cooltey) [inline app/src/main/res/layout/fragment_reading_lists.xml:61]: Forgot to mention but should this title also be bolded?

Comment (sharvaniharan) [reply app/src/main/res/layout/fragment_reading_lists.xml:61]: Done

**Pull request merged**